web/src/help_menu.js

```diff
@@ -0,0 +1,52 @@
+import $ from "jquery";
+
+import render_help_menu from "../templates/help_menu.hbs";
```

Comment:
I think `navbar_help_menu` might be a better name for this module and template; `informational_overlays` will probably get renamed at some point and I don't want this to sound too similar to that, so more specific will be good.

Reply:
Yeah!